### PR TITLE
Deposit files error

### PR DESCRIPTION
**Pull request by cwulfman**
Tests for ocr? and hocr? were broken because not dealing with empty string.

### CODE REVIEW

File: spec/exporters/hathi/content_package_spec.rb [hunk 1]
```diff
@@ -104,6 +104,17 @@
       expect(page.derivative_path.ftype).to eq "file"
     end
 
+    it "has a copy of the original image" do
+      image = package.pages.first.original_image
+      expect(image.mime_type).to eq "image/tiff"
+    end
+
+    # fix this: figure out how to get FactoryBot to create derivatives
```
Comment (escowles) [inline spec/exporters/hathi/content_package_spec.rb]: I believe you can use the `run_real_derivatives` directive like https://github.com/pulibrary/figgy/blob/master/spec/adapters/bagit/bag_exporter_spec.rb#L19

Comment (cwulfman) [reply spec/exporters/hathi/content_package_spec.rb]: Not sure what it's doing; the mime_type is still image/tiff, but I'd expect it to be image/jp2

**Comment by escowles** [comment]
@cwulfman That's very strange — I assume it's generating derivatives correctly when you run it instead of them being TIFFs?

**Comment by cwulfman** [comment]
That's right; they seem to be jp2s.

**Comment by hackmastera** [comment]
it's probably https://github.com/pulibrary/figgy/blob/master/spec/support/stub_tika.rb#L23

**Comment by cwulfman** [comment]
Well that would certainly explain it.  Do we care about this enough to start mucking around with this mocking code?  Or is there another way?

**Comment by escowles** [comment]
I don't think it's important to test that here, I think you can just remove the mime type check and this is fine.

**Pull request merged**